reactor-core/src/test/java/reactor/HooksTraceTest.java

```diff
@@ -376,7 +376,7 @@ public void onComplete() {
 		StepVerifier.create(ParallelFlux.from(Mono.just(1), Mono.just(1))
 		                                .log(null, Level.OFF)
 		                                .log(null, Level.OFF))
-		            .expectNext(7, 7) //from now counts as an additional one
+		            .expectNext(6, 6)
```

Comment:
why this has changed? How is that related?

Reply:
Before, the `Mono.just(1)` would be unnecessarily wrapped into a `Flux` using `Flux.from()` for no reason and an additional assembly hook would have been applied. Now, the `Mono` is identified as an assembled instance that doesn't need unnecessary wrapping using `Operators.toFluxOrMono()`.

Reply:
this is behaviour change. this has to be reverted